vmdb/app/controllers/application_controller/filter.rb

```diff
@@ -793,7 +797,11 @@ def adv_search_button
       if ["delete","saveit"].include?(params[:button])
         if @edit[:in_explorer]
           tree = x_active_tree.to_s
-          page.replace_html("#{tree}_div", :partial=>"vm_common/#{tree}")
+          if %w(cs_filter_tree).include?(tree)
```

Comment:
This can just be `if "cs_filter_tree" == tree`.  We don't need the extra arrays and method calls.